app/src/componentLibrary/breadcrumbs/breadcrumb/breadcrumb.jsx

```diff
@@ -14,32 +14,37 @@
  * limitations under the License.
  */
 
-import React, { useEffect, useRef } from 'react';
+import React, { useEffect, useRef, useState } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import { NavLink } from 'components/main/navLink';
+import { withTooltip } from 'componentLibrary/tooltip';
 import styles from './breadcrumb.scss';
 
 const cx = classNames.bind(styles);
 
-// todo remove title and add tooltip with {title} (EPMRPP-79184)
-export const Breadcrumb = ({
+const BreadcrumbTitleComponent = ({
+  descriptor: { link, title, onClick },
   maxBreadcrumbWidth,
   titleTailNumChars,
-  descriptor: { title, link, onClick },
+  tooltipDisabled,
+  setTooltipDisabled,
 }) => {
   const ref = useRef();
 
   useEffect(() => {
-    const { offsetWidth, scrollWidth, dataset } = ref.current;
+    if (ref.current) {
+      const { offsetWidth, scrollWidth, dataset } = ref.current;
 
-    if (offsetWidth < scrollWidth) {
-      dataset.tail = title.slice(title.length - titleTailNumChars);
+      if (offsetWidth < scrollWidth) {
+        dataset.tail = title.slice(title.length - titleTailNumChars);
+        setTooltipDisabled(false);
+      }
     }
-  }, [title, titleTailNumChars]);
+  }, [ref, title, titleTailNumChars]);
 
   return (
-    <div className={cx('breadcrumb')} title={title}>
+    <div className={cx(tooltipDisabled ? 'breadcrumb' : 'breadcrumb-with-tooltip')}>
```

Comment:
`breadcrumb` and `breadcrumb-with-tooltip` the same can we use only one of them?

Reply:
Unfortunately, no, it was made for correct work of ::before selector